applets/clock/clock.c

```diff
@@ -1132,6 +1132,7 @@ create_cities_section (ClockData *cd)
         GList *node;
         ClockLocationTile *city;
         GList *cities;
+        GList *l;
```

Comment:
You don't need to use a double-linked list since you only use `next` pointer, instead you can use a [GSList](https://developer.gnome.org/glib/stable/glib-Singly-Linked-Lists.html#GSList).

Reply:
Ditto.